automation/build-bin.ts

```diff
@@ -41,6 +41,13 @@ export const packageJSON = loadPackageJson();
 export const version = 'v' + packageJSON.version;
 const arch = process.arch;
 
+// DEBUG set to falsy for negative values else is truthy
+process.env.DEBUG = ['0', 'no', 'false', '', undefined].includes(
+	process.env.DEBUG?.toLowerCase(),
+)
+	? ''
+	: '1';
```

Comment:
`automation/run.ts` executes before `automation/build-bin.ts`, so `run.ts` is closer to being an entrypoint. It does created subprocesses, but if I recall correctly, the environment variables are inherited by the child processes.
Another separate entrypoint (executed in a `package.json` script) is `automation/capitanodoc/index.ts`, although I haven't seen occurrences of `process.env.DEBUG` there.

Reply:
Thanks moved to `automation/run.ts`